fed/mosaic-sns/src/main/resources/CSnsScheme.json

```diff
@@ -1,100 +1,155 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "CSnsScheme",
+    "description": "Schema describing the JSON file structure for the sns configuration.",
     "type": "object",
     "properties": {
-        "maximumTtl": { "type": "integer" },
-        "singlehopRadius": { "type": "number" },
+        "maximumTtl": { 
+            "description": "The threshold for the amount of hops for a transmission. This is an additional, hard threshold, which can't be exceeded.",
+            "type": "integer",
+            "default": 10
+        },
+        "singlehopRadius": { 
+            "description": "Default radius to be used if an AdhocConfiguration doesn't specify a radius.",
+            "type": "number",
+            "default": 509.4 
+        },
         "adhocTransmissionModel": {
-            "type": "object",
-            "properties": {
-                "type": { "type": "string" },
-                "simpleMultihopDelay": { "$ref": "#/definitions/delay" },
-                "simpleMultihopTransmission": { "$ref": "#/definitions/transmission" }
-            }
+            "description": "Defines the adhoc transmission model to be used for transmission.",
+            "$ref": "#/definitions/adhocTransmissionModel"
+        },
+        "singlehopDelay": { 
+            "description": "Delay configuration for a single hop.",
+            "$ref": "#/definitions/delay" 
         },
-        "singlehopDelay": { "$ref": "#/definitions/delay" },
         "singlehopTransmission": {
+            "description": "Transmission configuration for a single hop.",
             "$ref": "#/definitions/transmission"
         }
     },
     "definitions": {
+        "adhocTransmissionModel": {
+            "title": "adocTransmissionModel",
+            "description": "Object to define an adhoc transimssion model.",
+            "type": "object",
+            "properties": {
+                "type": { 
+                    "description": "Type of the transmission model. The simple model assumes, that all receivers can be reached even though this might not be the case in reality.",
+                    "type": "string",
+                    "enum": [ "simple","sophisticated" ]
+                },
+                "simpleMultihopDelay": {
+                    "description": "Delay to be used for a simple multihop.",
+                    "$ref": "#/definitions/delay" 
+                },
+                "simpleMultihopTransmission": { 
+                    "description": "Transmission parameters to be used for a simple multihop.",
+                    "$ref": "#/definitions/transmission" 
+                }
+            }
+        },
         "delay": {
```

Comment:
Please check for all default values of all delay properties.

Reply:
The delay properties don't have default values. Unless you mean the attributes that are of type "delay". They are set to have constantDelay as default. So I would add that and accordingly adapt the jsonscheme2md script